src/illuminatio/host.py

```diff
@@ -3,9 +3,14 @@
 """
 from abc import ABC
 
+import re
+
 import kubernetes as k8s
 
 
+ip_regex = re.compile(r"[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}")
```

Comment:
I would prefer to use https://docs.python.org/3/library/ipaddress.html instead of creating a custom regex which only works for IPv4 addresses.

Reply:
I reworked it, please take another look.

Reply:
LGTM